packages/karma-puppeteer-client/src/client.js

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-;(function (global) {
-  'use strict'
+(function (global) {
+  'use strict';
```

Comment:
Are these code changes from the linter?

Reply:
Yes.

Due to different folder structure these files were inadvertently ignored from the linter before. Once I moved them the linter chimed in.